test/unit/ssz_ex_test.exs

```diff
@@ -12,6 +12,493 @@ defmodule Unit.SSZExTest do
     assert {:error, ^error_message} = SszEx.decode(serialized, schema)
   end
 
+  test "packing a list of uints" do
+    list_1 = [1, 2, 3, 4, 5]
+
+    expected_1 =
```

Comment:
How did you compute these expected values?

Reply:
Packing is an intermediate step so I just tested with the values I would expect.

For correctness, the `hash_tree_root` of a list works which means this is right and it's also quite straightforward to compute.